wmn-data.json

```diff
@@ -2636,13 +2670,18 @@
        },
        {
         "name" : "Holopin",
-        "uri_check" : "https://holopin.io/@{account}#",

```

Comment:
I see this redirects to https://www.holopin.io as you found.

See above note for GETs vs POSTs. I think this one can be reworked to a working GET request by changing the `uri_check` to `https://www.holopin.io/@{account}`. Please confirm.

Reply:
Please revert to a GET.